src/components/RescueDetails.tsx

```diff
@@ -111,16 +114,49 @@ export default function RescueDetails({id}: { id: string }) {
                 setError('Failed to update rescue status. Please try again.')
             }
             setIsLoading(false)
-            if (newStatus !== "In Route") {
-                //setSelectedRescue(null)
-            }
         }
 
     }
 
+    ///// handle the change of the bird status
+    const handleBirdStatusChange = async (newBirdStatus : BirdStatus) => {
+
+        if (birdRescue) {
+            try {
+
+                let updatedFields
+                if (newBirdStatus === "Assessed" || newBirdStatus === "Died" || newBirdStatus === "No Show") {
+                    let newVolunteerStatus = "Incomplete"
```

Comment:
I think we talked about sending a bird back to Pending if it was only assessed

Reply:
I had meant to ask you about that as I couldn't remember what we had decided on. Would we want to also then remove the current volunteer? Or maybe add a note of some sort to explain the situation? Because currently it does not show the birdStatus on the frontend, only the volunteer status. Maybe that is something we can add, so if it's changed to assessed it will show a badge saying "Assessed"

Reply:
I'm not 100% sure where we landed either. Maybe they'll just see it on their kanban board and send a new alert?

Reply:
Yeah that may be one of the scenarios best left to be handled manually, at least for now. Next time we meet with them we could discuss if they would prefer a different workflow in that specific case.